libs/components/config/src/lib/config.ts

```diff
@@ -55,6 +60,7 @@ export interface RuntimeConfig {
   skyuxPathAlias: string;
   srcPath: string;
   useTemplateUrl: boolean;
+  localDev?: RuntimeConfigLocalDev;
```

Comment:
We mistakenly left off "Sky" from the `RuntimeConfig` interface; could we add it to yours since it's exported? `SkyRuntimeConfigLocalDev` Also, what do you think about putting `@experimental` above the type so we have an escape hatch for changes?

Reply:
Good call. I incorporated both changes.